src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -173,7 +173,19 @@ internal void AddChild(UIElement child, int? index = null)
 
 		internal void MoveChildTo(int oldIndex, int newIndex)
```

Comment:
We should do nothing if `oldIndex == newIndex`, instead of unnecessarily removal/insertion.

Reply:
That's done above in the UIElementCollection itself 👍